docs: enable section indexes (#11371)
PrairieLearn/PrairieLearn: docs/instructor-guide/index.md

# Instructor Guide

This section provides detailed documentation for all aspects of using PrairieLearn as an instructor.

> To run and test question development locally, see the [local installation guide](../installing.md).

Reviewer: ```suggestion
To develop questions and other course content on your own computer, see the [local installation guide](../installing.md).
```

> For more details about configuring the four main aspects of your course:

Reviewer: ```suggestion
Learn more about configuring the four main aspects of your course:
```

- [Courses](../course/index.md)

> - [Course Instances](../courseInstance.md)

Reviewer: ```suggestion
- [Course instances](../courseInstance.md)
```

- [Assessments](../assessment/index.md)
- [Questions](../question.md)

!!! tip

    For a refresher of what these aspects are, see the [PrairieLearn concepts](../concepts/index.md).

Some other important links are:

> - [Access Control](../accessControl/index.md)

Reviewer: Can you add a colon and some brief blurbs after each of these? E.g.:

```suggestion
- [Access control](../accessControl/index.md): learn how to control when and where students can access your course content.
```

Author: Resolved in https://github.com/PrairieLearn/PrairieLearn/pull/11371/commits/50b8c72f43aca12c6f7f3f65dd203378ad26a914.

> - [Element catalogue](../elements.md)

Reviewer: We typically use American English spellings (sorry Canadian friends):

```suggestion
- [Element catalog](../elements.md)
```

> - [Autograders](../externalGrading.md)

Reviewer: Most PL questions are technically auto-graded 🙂 

```suggestion
- [Code autograders](../externalGrading.md)
```

  - [Python](../python-grader/index.md)
  - [C / C++](../c-grader/index.md)
- [FAQ](../faq.md)

> - [syncing in-browser vs git](../sync.md)

Reviewer: I would just use the title of the actual page, "Editing and syncing course content".

Author: Resolved in https://github.com/PrairieLearn/PrairieLearn/pull/11371/commits/50b8c72f43aca12c6f7f3f65dd203378ad26a914.

> - [Element catalog](../elements.md): a reference for all elements available for use in questions
> - [Code autograders](../externalGrading.md): learn about the external code autograders
>   - [Python](../python-grader/index.md)
>   - [C / C++](../c-grader/index.md)
> - [FAQ](../faq.md): Common questions and answers
> - [Editing and syncing course content](../sync.md): Tradeoffs between in-browser editing and local editing

Reviewer: Capitalization/punctuation consistency nits. Also includes a reference to the Java grader.

Maybe include links to various bits of workspace documentation too? Those are a pretty fundamental PL feature and differentiator.

```suggestion
- [Element catalog](../elements.md): a reference for all elements available for use in questions.
- [Code autograders](../externalGrading.md): learn how to automatically grade student code.
  - [Python](../python-grader/index.md)
  - [C / C++](../c-grader/index.md)
  - [Java](../java-grader/index.md)
- [FAQ](../faq.md): find answers to common questions.
- [Editing and syncing course content](../sync.md): understand tradeoffs between in-browser editing and local editing.
```
